DESCRIPTION

```diff
@@ -72,4 +74,4 @@ Collate:
     'helpers.R'
     'azimuth.R'
     'generics.R'
-    'server.R'
+    'server.R'
```

Comment:
nit—you should include a blank line at the end of files:

```suggestion
    'server.R'
    
```

 It's probably worth you taking a read through [Google's R Style Guide ](https://google.github.io/styleguide/Rguide.html).